### PR TITLE
OpenSUSE - helix amd64

### PR DESCRIPTION
**Pull request by aik-jahoda**
Adds OpenSUSE 15.02 Amd64 for the Helix.

Part of #502

/cc: @wfurt

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -1354,6 +1354,19 @@
               }
             }
           ]
+        },
+        {
+          "platforms": [
+            {
+              "architecture": "amd64",
+              "dockerfile": "src/opensuse/18.04/helix/amd64",
```
Comment (mthalman) [inline manifest.json:1362]: Shouldn't the directory be named 15.2 instead of 18.04?

File: src/opensuse/18.04/helix/amd64/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        libunwind \
+        lldb-devel \
+        gcc-locale \
+        python3-devel \
+        python3-pip \
+        sudo \
+        unzip \
+        glibc-i18ndata && \
+        \
+        zypper clean -a
```
Comment (mthalman) [inline src/opensuse/18.04/helix/amd64/Dockerfile:30]: Some of the package names are not in alphabetical order.  Please re-order them.

**Review by mthalman — COMMENTED** [review]
This will require a new pipeline YAML file.  You can make a copy of the two YAML files for an existing distro and update accordingly.  For example: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/main/eng/pipelines/dotnet-buildtools-prereqs-fedora-pr.yml and https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/main/eng/pipelines/dotnet-buildtools-prereqs-fedora.yml.  Then just find-and-replace "fedora" with "opensuse".

**Review by mthalman — APPROVED** [review]

File: manifest.json [hunk 3]
```diff
@@ -1354,6 +1354,19 @@
...
             }
           ]
+        },
+        {
+          "platforms": [
+            {
+              "architecture": "amd64",
+              "dockerfile": "src/opensuse/15.2/helix/amd64",
+              "os": "linux",
+              "osVersion": "leap",
```
Comment (mthalman) [inline manifest.json:1364]:
```suggestion
              "osVersion": "leap15.2",
```

And then I'll need to make another change to Image Builder to support a display name for this.

Comment (mthalman) [reply manifest.json:1364]: https://github.com/dotnet/docker-tools/pull/874

**Comment by aik-jahoda** [comment]
@mthalman, the build fails on 

```
INGESTING IMAGE INFO DATA INTO KUSTO
------------------------------------
Unhandled exception: System.NotSupportedException: The OS version 'leap15.2' is not supported.
```

Should be following change leap15.2?
https://github.com/dotnet/docker-tools/commit/6377dc0e909aaa0bc6cdf2c038f0cd5bbc8c8560

Or should I revert ```"osVersion": "leap15.2"``` to ```"osVersion": "leap"``` in manifest.json?

**Comment by mthalman** [comment]
> Or should I revert "osVersion": "leap15.2" to "osVersion": "leap" in manifest.json?

Nope, this is what https://github.com/dotnet/docker-tools/pull/874 is for.  I'm working on getting that integrated into this repo.  I'll merge this PR once things are working.

Looks like the manifest.json files needs some conflicts resolved too.

**Comment by aik-jahoda** [comment]
Conflicts were resolved

**Comment by mthalman** [comment]
Published image metadata: https://github.com/dotnet/versions/commit/775adb8c39df753f79b91804d5193615f9806108

**Pull request merged**